<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zippanel</class>
 <widget class="QWidget" name="zippanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>zip</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5">
   <item>
    <widget class="QGroupBox" name="groupBtn">
     <property name="title">
      <string>Pack Upload</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnServal">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Select Serval File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnZipTool">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Select 7z.exe File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkUsing7z">
        <property name="text">
         <string>Use Internal Zipfile tool instead</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCut">
        <property name="text">
         <string>Cut Serval File into Subfiles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Select Save Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStart">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Start Packing</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="txtOut"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
